dnf/repo.py

```diff
@@ -565,6 +565,10 @@ def load(self):
         try:
             ret = self._repo.load()
         except RuntimeError as e:
+            if self.skip_if_unavailable:
+                logger.warning(str(e))
```

Comment:
I believe that error message is without repo ID. Am I correct?

Reply:
In most cases, it's this message from libdnf: ``"Failed to synchronize cache for repo '<repo-id>'"``.
PR https://github.com/rpm-software-management/libdnf/pull/661 adds the same message also for GPG keys.
But for unexpected error, the repo ID is missing, so yes, that might be problematic.